src/view.c

```diff
@@ -921,6 +934,7 @@ view_destroy(struct view *view)
 
 	/* Remove view from server->views */
 	wl_list_remove(&view->link);
+	free(view->ssd);
```

Comment:
Having `ssd_destroy()` not actually destroy the SSD seems a bit strange on the first look. But I think we have a similar pattern with `xxx_finish()` and `xxx_destroy()` somewhere else already. Not sure right now if its `theme.c` or `rcxml.c`. We should likely decide on one naming convention, add it to Contributing.md and stick with it.

The straight `free()` call seems wrong to me though, `ssd_free()` (or a better named variant) would be more intuitive IMHO because we didn't `calloc()` / `znew()` it ourselves but instead called `ssd_create()` in `xdg.c` / `xwayland.c`.

Reply:
I'll add `ssd_free()`, it seems like a little overkill now, but I agree it's consistent and is good future-proofing.

Reply:
Added `ssd_free()`

Reply:
Removed `ssd_free()` again and made `ssd_destroy()` actually free the `struct ssd`